pkg/lib/cockpit-components-table.jsx

```diff
@@ -161,6 +164,7 @@ export class ListingTable extends React.Component {
         // We need the following because of: https://github.com/patternfly/patternfly-react/issues/3090
         if (props.cells.some(cell => typeof (cell.title) != 'string' || cell.title.toLowerCase() == 'id' || !cell.title))
             tableBodyProps.rowKey = ({ rowData, rowIndex }) => rowIndex;
```

Comment:
Using an ID for each row seems strange, though. This is the place that sets the keys for the rows, and obviously rowIndex is an useless value for a row key. IMHO it would be better to make `rowKey` a mandatory property of a table row component, and let the caller set it. Perhaps it could have a default like a hash value of `rowData`, but I'm not sure if that's possible/easy to do in JS.